app/ui/legacy/src/main/java/com/fsck/k9/ui/messagelist/MessageListAdapter.kt

```diff
@@ -51,10 +51,10 @@ class MessageListAdapter internal constructor(
     private val relativeDateTimeFormatter: RelativeDateTimeFormatter,
 ) : RecyclerView.Adapter<MessageListViewHolder>() {
 
-    private val forwardedIcon: Drawable = ResourcesCompat.getDrawable(res, R.drawable.ic_messagelist_forwarded, theme)!!
-    private val answeredIcon: Drawable = ResourcesCompat.getDrawable(res, R.drawable.ic_messagelist_answered, theme)!!
-    private val forwardedAnsweredIcon: Drawable = ResourcesCompat
-        .getDrawable(res, R.drawable.ic_messagelist_answered_forwarded, theme)!!
+    private val forwardedIcon: Drawable = ResourcesCompat.getDrawable(res, Icons.Outlined.Forward, theme)!!
+    private val answeredIcon: Drawable = ResourcesCompat.getDrawable(res, Icons.Outlined.Reply, theme)!!
+    private val forwardedAnsweredIcon: Drawable =
+        ResourcesCompat.getDrawable(res, Icons.Outlined.CompareArrows, theme)!!
```

Comment:
These are not regular icons. They were (somewhat) optimized for being displayed in line with text in the message list.

|Before|After|
|-|-|
|![image](https://github.com/thunderbird/thunderbird-android/assets/218061/62d1ff6b-c2b8-4c0c-be27-58d7f78f1526)|![image](https://github.com/thunderbird/thunderbird-android/assets/218061/3e999b3c-28b7-4579-9fd9-9122b6b93812)|

I think it would be best to keep the old icons for now and ask the design team to come up with new icons (and a color to use).

Reply:
They should be colored using a tint instead of coloring the icon directly as done previously. The icons themselfs need design another discussion. I'll undo the changes until there is a good replacement.

Reply:
I had a look at the latest gmail app and they use the new icons in the detail screen and the old ones in the list. But the tint is the same as `onSurface`. I created a patch to align the icon.

<img width="541" alt="grafik" src="https://github.com/thunderbird/thunderbird-android/assets/2285235/8493ecc3-baab-4095-ae0b-315620b17879">